tutorials/parser/pom.xml

```diff
@@ -275,6 +275,18 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>com.univocity</groupId>
+      <artifactId>univocity-parsers</artifactId>
+      <version>2.0.0</version>
+      <optional>true</optional>
```

Comment:
This dependency is only used for certain features in the project, and will not be needed if that feature isn't used. (On the lines of super-csv dependency)